unlock-app/src/components/interface/connect/UnlockAccount.tsx

```diff
@@ -228,15 +166,13 @@ const SignUp = ({ email, onReturn, signUp, onSignIn }: SignUpProps) => {
     <div className="grid gap-2">
       <form className="grid gap-4 px-6" onSubmit={handleSubmit(onSubmit)}>
         <Input
```

Comment:
I think we don't need to show this. Let's just show something like "Please create your account"